src/Microsoft.CSharp/src/MultilingualResources/Microsoft.CSharp.de.xlf

```diff
@@ -0,0 +1,520 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I didn't notice a major impact on build time, but I haven't actually timed it.  For the official builds, where we want localization, I don't see a way that we could avoid this impact (even if it is significant).  I think that the approach used in this PR, with the .xlf and localized .resx files directly in the project, would be the most performant approach.  That's actually another downside to the approach of dynamically generating machine translations with the build; when I did it for just the Microsoft.CSharp project, it took somewhere around 30 seconds.  I think putting them in a separate repo and dynamically compiling them into the project with the build would add significant time to the build also.

One thing that I think we should consider is supporting the ability to not build localization when building locally, in order to reduce build time.  I was thinking that that would be something that we would consider after getting the initial localization infrastructure in place, because it's lower priority.